main.go

```diff
@@ -147,6 +146,7 @@ func main() {
 
 	tokenManager := token.NewManager(publicKey, privateKey)
 	app.UseJWTMiddleware(service, jwt.New(publicKey, nil, app.NewJWTSecurity()))
+	service.Use(login.ConfigureExtractUser(tokenManager))
```

Comment:
JWT security runs as a handler right before the Controller function is invoked. We can never get the token via a middleware. 

login.ContextIdentity(ctx) needs to do this work.